client/src/ingredients/browser-compatibility-table/browser-support-notes.js

```diff
@@ -75,7 +88,7 @@ export function BrowserSupportNotes({ indexNote, blockElementTag, noteElementTag
   if (indexNote.alternatives.length > 0) {
     browserSupportNotes.push(
       indexNote.alternatives.map((alternative, index) => {
-        currentNoteContent = `<abbr class="only-icon" title="Uses the non-standard name: <code>${alternative.alternative_name}</code>"><span>Alternate Name</span><i class="ic-altname"></i></abbr> Uses the non-standard name: <code>${alternative.alternative_name}</code>`
+        currentNoteContent = <><abbr className="only-icon" title={`Uses the non-standard name: <code>${alternative.alternative_name}</code>`}><span>Alternate Name</span><i className="ic-altname"></i></abbr> Uses the non-standard name: <code>{alternative.alternative_name}</code></>
```

Comment:
HTML tags in a tooltip? Was this deliberate? (not a security thing, but still a bit weird)

Reply:
I actually took this from an example at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio, but you're right..that is weird..

Reply:
No, that is not correct. The renderer shouldn't need to do that kind of work. That's what the `build-json` script in `stumptown-content` is supposed to own. 
If the BCD content in the JSON files that it picks up from `stumptown/packaged/**/*.json` isn't right, then let's hop over to https://github.com/mdn/stumptown-content/issues/new and resolve it there. 
File the issue and to avoid losing steam come back here and write in the issue URL as a code comment. 

Right, @wbamberg ?

Reply:
Oops. Posted this response in the wrong place. See above.

Reply:
I removed the tags from the tooltip.

Reply:
From what I can tell, BCD would like to remove these things in the source, too: https://github.com/mdn/browser-compat-data/issues/4033 .

Reply:
Ah i see, yeah i tried to keep all of this in-line with what is currently out there in the wild. Makes sense to remove it though.